Dockerfile-debian.template

```diff
@@ -16,26 +16,25 @@
 # limitations under the License.
 ###############################################################################
 
-FROM openjdk:8-jre
+FROM openjdk:8-jre-slim
+
+# Install dependencies
+RUN set -ex; \
+  apt-get update; \
+  apt-get -y install gnupg libsnappy1v5 wget; \
+  rm -rf /var/lib/apt/lists/*
 
 # Grab gosu for easy step-down from root
 ENV GOSU_VERSION 1.7
 RUN set -ex; \
   wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$(dpkg --print-architecture)"; \
   wget -O /usr/local/bin/gosu.asc "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$(dpkg --print-architecture).asc"; \
-  export GNUPGHOME="$(mktemp -d)"; \
```

Comment:
oups you got me, I was annoyed because I was having one exception removing this temporal directory, I send a cleaner version in secs.